atomic_reactor/inner.py

```diff
@@ -212,6 +212,13 @@ def add_docker_registries(self, registry_uris, insecure=False):
         for registry_uri in registry_uris:
             self.add_docker_registry(registry_uri, insecure=insecure)
 
+    def remove_docker_registry(self, docker_registry):
+        if docker_registry in self._registries["docker"]:
+            self._registries["docker"].remove(docker_registry)
+        else:
+            raise RuntimeError("Tried to remove docker registry outside of the list: %s"
+                               % self._registries["docker"].keys())
```

Comment:
The `else` block doesn't get covered in unit testing.

Not sure if we need to do this checking anyway. Won't the `KeyError` that would get raised be sufficient?
```
def remove_docker_registry(self, docker_registry):
    self._registries['docker'].remove(docker_registry)
```